ts/lib/commands/precompile.ts

```diff
@@ -36,6 +38,24 @@ export default command({
 
     let manifestPath = options.manifestPath;
     let packageName = this.project.pkg.name;
+
+    // Ensure that if we are dealing with an addon that is using a different
+    // addon name from its package name, we use the addon name, since that is
+    // how it will be written for imports.
+    if (this.project.isEmberCLIAddon()) {
```

Comment:
Since we only ever expose the precompile command to addons, this should always be `true` shouldn't it?

Reply:
Ah, good point.